legate/core/context.py

```diff
@@ -226,6 +252,25 @@ def get_sharding_id(self, shard_id: int) -> int:
     def get_tunable(
         self, tunable_id: int, dtype: DataType, mapper_id: int = 0
     ) -> npt.NDArray[Any]:
+        """
+        Queries a tunable parameter to the mapper.
+
```

Comment:
I'll defer the description on tunable parameters to a follow-on programming model doc. there are two sides of tunable parameters clients need to fill in and it may not be obvious without enough context why we separate these.